src/main/scala/devices/spi/SPIBundle.scala

```diff
@@ -93,6 +94,8 @@ object SPIControl {
     ctrl.dla.interxfr := UInt(0)
     ctrl.wm.tx := UInt(0)
     ctrl.wm.rx := UInt(0)
+    ctrl.extradel.coarse := UInt(2)
```

Comment:
will the value `2` always work out of reset? For any value of `sckdiv`? I mean, is 2 the safest value possible?

Reply:
Yes, 2 will work always ..it is presently hardcoded.